File: Z.ExtensionMethods-master/src (VB.NET)/Z.ExtensionMethods.VB/Z.Core/System.Int64/GenericInt/Int64.Milliseconds.vb

' Copyright (c) 2015 ZZZ Projects. All rights reserved
' Licensed under MIT License (MIT) (https://github.com/zzzprojects/Z.ExtensionMethods)
' Website: http://www.zzzprojects.com/
' Feedback / Feature Requests / Issues : http://zzzprojects.uservoice.com/forums/283927
' All ZZZ Projects products: Entity Framework Extensions / Bulk Operations / Extension Methods /Icon Library


Public Module Extensions_306

	''' <summary>
	'''     An Int64 extension method that milliseconds the given this.
	''' </summary>
	''' <param name="this">The @this to act on.</param>
	''' <returns>A TimeSpan.</returns>
	<System.Runtime.CompilerServices.Extension> _
	Public Function Milliseconds(this As Int64) As TimeSpan
		Return TimeSpan.FromMilliseconds(this)
	End Function
End Module

File: Z.ExtensionMethods-master/src/Z.Core/System.Int64/GenericInt/Int64.Milliseconds.cs
using System;

public static partial class Extensions
{
    /// <summary>
    ///     A long extension method that converts the given value to a TimeSpan representing the specified number of milliseconds.
    /// </summary>
    /// <param name="this">The value to act on, representing milliseconds.</param>
    /// <returns>A TimeSpan representing the specified number of milliseconds.</returns>
    public static TimeSpan Milliseconds(this long @this)
    {
        return TimeSpan.FromMilliseconds(@this);
    }
}
